.github/workflows/tests.yml

```diff
@@ -316,7 +324,7 @@ jobs:
           path: ${{matrix.workdir || github.workspace}}/bottles
 
       - name: Post cleanup
-        if: always()
+        if: ${{ !matrix.ephemeral && always() }}
```

Comment:
Don't need `${{ ... }}` I think.

(Example: da6187b2853041a68613b7deab53cead15fdf7e5)

Reply:
Ah, yea, I don't need it if `always()` comes first.